cypress/testcases/rancher/rancher_integration.spec.ts

```diff
@@ -0,0 +1,363 @@
+    },
+    rke1_cluster_attributes: {
+        cpus: '',
+        memory: '',
+        disk: '',
+        namespace: '',
+        image: '',
+        network_name: '',
+        ssh_user: '',
+        rke1_latest: '',
+        rke1_stable: '',
+        user_data_template: ''
+    }
+};
+
+/**
+ * 1. Create image with cloud image available for openSUSE. http://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.3/images/openSUSE-Leap-15.3.x86_64-NoCloud.qcow2
+ * 2. Click save
+ * 3. Try to edit the description
+ * 4. Try to edit the URL
+ * 5. Try to edit the Labels
+ * Expected Results
+ * 1. Image should show state as Active.
+ * 2. Image should show progress as Completed.
+ * 3. User should be able to edit the description and Labels
+ * 4. User should not be able to edit the URL
+ * 5. User should be able to create a new image with same name.
+ */
+describe('Rancher Integration Test', function() {
+
+    const imageEnv = Cypress.env('image');
+    // const IMAGE_NAME = imageEnv.name;
+    // const IMAGE_URL = imageEnv.url;
+    const IMAGE_NAME = 'focal-server-cloudimg-amd64.img';
+    const IMAGE_URL = 'https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img';
+    
+    const value = {
+        name: IMAGE_NAME,
+        url: IMAGE_URL,
+    }
+
+    beforeEach(() => {
+        cy.fixture('rancher').then((data) => {
+            rData = data;
+
+        });
+    })
+
+    it('Prepare Harvester Image', () => {
+        cy.login();
+
+        // create IMAGE according to the value set
+        image.goToCreate();
+        image.create(value);
+        image.checkState(value);
+
+    });
+
+    it('Prepare Harvester VLAN network', () => {
+        cy.login();
+
+        network.createVLAN('vlan1', 'default', '1', 'mgmt')
+
+    });
+
+    it('Rancher import Harvester', { baseUrl: constants.rancherUrl}, () => {
+        // cy.login();
+        cy.visit('/');
+        
+        rancher.rancherLogin();
+        
+        rancher.importHarvester().then((el) => {
+            let copyImportUrl = el.text();
+            console.log('Get: ', copyImportUrl)
+            cy.log(copyImportUrl);
+            cy.task('setGlobalVariable', copyImportUrl)
+        }).as('importCluster');
+
+    });
+
+
+    it('Harvester import Rancher', () => {
+        cy.login();
+        rancher.registerRancher();
+    });
+
+    
+    it('Check Harvester Cluster Status', { baseUrl: constants.rancherUrl}, () => {
+        // cy.login();
+        cy.visit('/');
+        cy.wait(5000);
+
+        rancher.rancherLogin();
+        
+        rancher.visit_virtualizationManagement();
+
+        rancher.checkState(rData.harvester_cluster_name);
+
+    });
+
+    it('Create Cloud Credential', { baseUrl: constants.rancherUrl}, () => {
+        // cy.login();
+        cy.visit('/');
+        cy.wait(5000);
+
+        rancher.rancherLogin();
+
+        rancher.createCloudCredential(rData.cloud_credential, rData.harvester_cluster_name);
```

Comment:
<img width="1693" alt="image" src="https://user-images.githubusercontent.com/24985926/201815692-1347ee7f-8add-4873-ab83-de7e32cc5073.png">

When the cluster is selected, the UI sends a request to get kubecfongContent, which is the cause of the failure

Reply:
The error may caused by the previous import Harvester failed or there is a harv-local cloud credential exists